src/components/addBooks.js

```diff
@@ -1,16 +1,34 @@
-import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
-const AddBooks = () => (
-  <form>
-    <input placeholder="ADD NEW BOOK" />
-    <select name="books" id="book">
-      <option value="Science fiction">The God&apos;s Equation</option>
-      <option value="Action">Volvo</option>
-      <option value="Economy">Dollars</option>
-      <option value="Computer Science">DS n ALgo</option>
-    </select>
-    <button type="submit">Add Book</button>
-  </form>
-);
+const AddBooks = () => {
+  const dispatch = useDispatch();
+
+  const submitBookToStore = (e) => {
+    const newBook = {
+      id: uuidv4(),
+      title: e.target.previousElementSibling.previousElementSibling.value,
+      author: '',
```

Comment:
- [ ] It will be great to have a local state for your title and author. This is a major requirement for this project. The values of both the title and the author should have from your local state. You can do this by using the use state hook. Kindly read this[ article ](https://reactjs.org/docs/hooks-state.html)for more explanation

Reply:
So you are pointing to a place where the value is from the screen to update the state, but the screen has no input field for author by the template. Please provide more info. T

Reply:
Done.

Reply:
This issue is well resolved! ✔️